Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -794,16 +794,17 @@ BeaconAPIEnabled:
 
 BlobRegistryTopOriginPartitioningEnabled:
   type: bool
-  status: unstable
+  status: testable
+  category: dom
```

Comment:
Maybe this should be `security` or `networking`. I don't really think of this as a DOM feature.